app/components/markdown/index.js

```diff
@@ -303,9 +307,38 @@ export default class Markdown extends PureComponent {
         );
     }
 
+    renderEditedIndicator = ({context}) => {
+        let spacer = '';
+        let opacity = Platform.select({ios: 0.7, android: 1});
+        if (context[0] === 'paragraph') {
+            spacer = ' ';
+            opacity = Platform.select({ios: 0.5, android: 0.6});
+        }
+        const styles = [style.editedIndicatorText, {opacity}];
+        if (Platform.OS === 'ios') {
+            styles.push(this.computeTextStyle(this.props.baseTextStyle, context));
+        }
+        return (<Text style={styles}>
+            <Text>{spacer}</Text>
```

Comment:
The extra `<Text>` around `{spacer}` may not be necessary

Reply:
it is, if we don't add it in a Text component RN will complain, I'm almost sure